### PR TITLE
minor fixes

### LINKED ISSUE #1638: Consume is not working
Asset: https://market.oceanprotocol.com/asset/did:op:a1b1fdca0ff4eaa99714a3a1021e703f0e74be709757944a3e983f8a7a4fca13

### PR DESCRIPTION
**Pull request by mihaisc**
- fixes #1638

### CODE REVIEW

File: src/@utils/order.ts [hunk 1]
```diff
@@ -57,7 +57,10 @@ export async function order(
     _consumeMarketFee: {
       consumeMarketFeeAddress: marketFeeAddress,
       consumeMarketFeeAmount: consumeMarketOrderFee,
-      consumeMarketFeeToken: asset.accessDetails.baseToken.address
+      consumeMarketFeeToken:
+        asset.accessDetails && asset.accessDetails.baseToken
```
[the remark this answers is not in the document]
Comment (mihaisc) [reply src/@utils/order.ts:61]: yes yes, this is messy , it's just for testing purposes

**Comment by EnzoVezzaro** [comment]
Not sure why, but the buy button takes a lot of time to get enabled on the details page: 

https://user-images.githubusercontent.com/13741335/183645171-14388570-204f-4600-9a32-6099ae05f368.mp4

**Comment by mihaisc** [comment]
Yes, it's waiting for the price to calculate. I'll check why that loading message is not there anymore

**Comment by mihaisc** [comment]
Feedback is appreciated on the message : `Calculating full price (including fees)`

**Comment by kremalicious** [comment]
> Not sure why, but the buy button takes a lot of time to get enabled on the details page:

yes, we had this, we need to wait for price & fees to load before button can be activated. I removed the loading state because in our current implementation it's impossible to get this right. Adding loading will show:

- disabled button
- loader
- enabled button

When all we should have is a button which gets activated once everything is fetched. The price itself then should get a loading indicator, not the button

**Comment by mihaisc** [comment]
Done, i wouldn't consider a bigger refactor now. It's more important to merge a fix.

File: src/components/Asset/AssetActions/Download.tsx [hunk 2]
```diff
@@ -83,7 +88,9 @@ export default function Download({
   }, [dtBalance])
 
   useEffect(() => {
+    console.log(orderPriceAndFees)
```
Comment (kremalicious) [inline src/components/Asset/AssetActions/Download.tsx:91]: missing console.log

**Review by kremalicious — APPROVED** [review]

File: src/components/Asset/AssetActions/Download.tsx (hunk 2 above)
Comment (mihaisc) [reply src/components/Asset/AssetActions/Download.tsx]: forgotten console.logs ... my bane :))

**Pull request merged**